templates/rspec_api_documentation/api_blueprint_index.mustache

```diff
@@ -16,34 +19,49 @@ FORMAT: 1A
 ## {{ route_name }} [{{ route }}]
 {{# description }}
 
-description: {{ description }}
+{{ description }}
```

Comment:
Not sure what value this should have... or if it even works.

Reply:
Coll, thanks, we just need to add apiblueprint validations in spec suite, and we'll be done.

Reply:
Fixed the specs, and added an expectation for the "route explanations". I can't seem to write an expectation for the "method explanations", although it works when I use the gem. Something seems off with the `http_method`s in the specs for some reason.